### DISCUSSION TITLE
Problems w video-only meetings:  random addresses used

### DISCUSSION THREAD

**Discussion by WMikeCarroll** [code4recovery/12-step-meeting-list#521] [Q&A]
If the Address field of a meeting is empty or contains a non-address, then the site grabs an address from someplace in my upload and shows that address for this meeting.  The site also shows that the meeting is a member of a large group of meetings with the same random address.  

If anything at all is in the Address field (such as "Video Only") in an attempt to disable this automatic address grab, then the page lists it as "In-person and Online" (which it is not).

I have attached the Excel file I convert to a CSV before uploading.  This file includes some fields not used by the WP site. 

I would appreciate any guidance.  Thanks 
Mike C
Here is a grab from the page showing a video only meeting with a random address.

![image](https://user-images.githubusercontent.com/90402795/132717503-a0f5f041-4cbc-4956-bae2-78fd97f08836.png)

This is a snip from the Excel file for that meeting.  I have hidden some columns so hopefully you can read it here.
![image](https://user-images.githubusercontent.com/90402795/132716803-14d5f892-8b9a-4571-b7e2-dd596601a6e6.png)

**Comment by tech2serve** [comment]
Hi Mike--

All addresses *must* geocode. In order to enter a meeting, you need to enter a valid address.

For meetings that cannot be attended in person (i.e., online only), please enter the city and state where the meeting is hosted, and then enter the online details in the appropriate fields.

For meetings that can be attended in person, you enter a full street, city and state address.

This should correct any issues you see.

Tim

**Reply by WMikeCarroll** [reply]
Hi there Tim,
Thanks so much for getting back so quickly.
Yes, I am following these guidelines.
Here is a snip - the meetings w/o addresses are online, as are the "Video Only".  Regardless, the page shows a strange address and puts the online meetings in large group. 
![image](https://user-images.githubusercontent.com/90402795/132730166-cb943b30-1e92-41fe-883f-cc8647902435.png)
The Antioch Umbrella group shown in my first chat is second from the top in this snip.

**Reply by WMikeCarroll** [reply]
I just put "Ignore the Map" in the Location field for all the video meetings.  That seems to have stopped the site from showing a page w a fictitious address.  
Good enough.  But we shall see, eh?
Thanks.
Mike C

**Reply by tech2serve** [reply]
I believe you misunderstood the guidelines. If you put anything in "Address" other than addresses that are geocodable by Google, things will not work as expected. Don't put "Video Only" in Address. Don't leave it blank. 

For a meeting that is online only, enter "Antioch, CA" (or some other city) in the Address field. A city and state is geocodable by Google and will work. 

For a meeting that is available in person, enter "123 Main Street, Antioch, CA" (or some other specific address). 

BTW, GSO's Meeting Guide *requires* geocodable addresses (which is why we require them) in order for the meeting to be imported into the mobile app.

**Comment by WMikeCarroll** [comment]
On a related issue, it seems that if a field is empty, that the web site substitutes some other information.  In my case, I don't upload Contact Information, and the site substitutes what I have in the Website field.  This feature - substituting one field into an empty field - has caught me a few time before.  Is this necessary?
Thanks.

**Reply by tech2serve** [reply]
@WMikeCarroll,

My sincere apologies for missing this additional comment on this thread. I have no excuse.

I will try and duplicate this, but my gut is telling me that something else on your site is causing the substitution. A lot of users import spreadsheets without filling in contact information and they haven’t raised an issue with random substitutions. If you’re still experiencing this, pleas open a dev/debug console on your browser and watch for errors when you do the import. It might also be helpful to set up a test site and work there if you can. That will help you isolate the issue.

If you get more information, please report back. If I’m able to duplicate, I’ll convert this to a bug report.

Thanks.

@code4recovery/twelve-step-meeting-list
